cupy/sorting/search.py

```diff
@@ -24,7 +24,28 @@ def argmax(a, axis=None, dtype=None, out=None, keepdims=False):
     return a.argmax(axis=axis, dtype=dtype, out=out, keepdims=keepdims)
 
 
-# TODO(okuta): Implement nanargmax
+def nanargmax(a, axis=None, dtype=None, out=None, keepdims=False):
+    """Return the indices of the maximum values in the specified axis ignoring
+    NaNs. For all-NaN slice ``-1`` is returned.
+    Subclass cannot be passed yet, subok=True still unsupported
+
+    Args:
+        a (cupy.ndarray): Array to take nanargmax.
+        axis (int): Along which axis to find the maximum. ``a`` is flattened by
+            default.
+
+    Returns:
+        cupy.ndarray: The indices of the maximum of ``a``
+            along an axis ignoring NaN values.
+
+    .. note:: For performance reasons, ``cupy.nanargmax`` returns ``-1`` for
```

Comment:
Sorry, could you fix the comment `-1` to `out of range values`?

Reply:
Okay done